Krypt/Source/SHA256.swift

```diff
@@ -20,4 +20,45 @@ public struct SHA256 {
     }
     return status == errSecSuccess ? Data(hash) : nil
   }
+
+  // MARK: - Buffered SHA-256 Calculation
+
+  /// Calculates SHA256 hash of a given file using a buffer to avoid running out of memory for potentially large files
+  /// - Parameters:
+  ///   - url: The url to the file to calculate the SHA256 hash for
+  ///   - bufferSize: The size of the buffer to use in bytes, defaults to 1024 * 1024 bytes =  1MB
+  public static func digest(file url: URL, with bufferSize: Int = 1024 * 1024) throws -> Data {
```

Comment:
```suggestion
  public static func digest(file url: URL, withBufferSize bufferSize: Int = 1024 * 1024) throws -> Data {
```
I think it still needs the `withBufferSize`, just "with" can be quite confusing with primitive types.